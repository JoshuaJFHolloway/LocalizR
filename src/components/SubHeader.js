import React from 'react';
import SubHead from "./styledComponents/subheader.js";

const SubHeader = () => {
  return (
    <SubHead>The adventure language learning game!</SubHead>
  );
};

export default SubHeader;