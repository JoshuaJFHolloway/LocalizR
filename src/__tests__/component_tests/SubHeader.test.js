import React from 'react';
import { shallow } from 'enzyme';
import SubHeader from '../../components/SubHeader';

describe('SubHeader', () => {
  let subheader = shallow(<SubHeader />);

  it('renders correctly', () => {
    expect(subheader).toMatchSnapshot();
  });

});
