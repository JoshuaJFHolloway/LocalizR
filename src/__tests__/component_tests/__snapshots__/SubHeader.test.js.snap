// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubHeader renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SubHeader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": " Your adventure language learning game :D ",
    },
    "ref": null,
    "rendered": " Your adventure language learning game :D ",
    "type": "h2",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": " Your adventure language learning game :D ",
      },
      "ref": null,
      "rendered": " Your adventure language learning game :D ",
      "type": "h2",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
